doc/neps/nep-0022-ndarray-duck-typing-overview.rst

```diff
@@ -0,0 +1,340 @@
+==========================================================
+NEP 22: duck typing for NumPy arrays – high level overview
+==========================================================
+
+:Author: Stephan Hoyer <shoyer@google.com>, Nathaniel J. Smith <njs@pobox.com>
+:Status: Draft
+:Type: Informational
+:Created: 2018-03-22
```

Comment:
we might as well update this :)

Reply:
eh, that is when it was created... we could bump to today if you want